packages/expo-updates-interface/android/src/main/java/expo/modules/updatesinterface/UpdatesInterface.java

```diff
@@ -30,5 +30,6 @@
     String getLaunchAssetPath();
   }
 
+  void reset();
```

Comment:
```suggestion
  void reset();
  
```
nit about spacing. Made it match the OBJ-C spacing in the analagous file.